periodic_tasks/common/settings.py

```diff
@@ -29,6 +29,7 @@
 MAX_FEE_PER_GAS: Wei = Web3.to_wei(MAX_FEE_PER_GAS_GWEI, 'gwei')
 PRIORITY_FEE_NUM_BLOCKS: int = config('PRIORITY_FEE_NUM_BLOCKS', default=10, cast=int)
 PRIORITY_FEE_PERCENTILE: float = config('PRIORITY_FEE_PERCENTILE', default=80.0, cast=float)
+GAS_LIMIT: Wei = config('GAS_LIMIT', default=0, cast=int)
```

Comment:
I guess the GAS_LIMIT is added here to fix the issue with the internal execution node. I think we can skip it as we assume the node returns proper gas params.